infra/base-images/base-runner/run_fuzzer

```diff
@@ -21,11 +21,20 @@
 export PATH=$OUT:$PATH
 cd $OUT
 
+DEBUGGER=${DEBUGGER:-}
+
 FUZZER=$1
 shift
 
 CORPUS_DIR="/tmp/${FUZZER}_corpus"
 
+SANITIZER=${SANITIZER:-}
+if [ -z $SANITIZER ]; then
+  # If $SANITIZER is not specified (e.g. calling from `reproduce` command), it
+  # is not important and can be set to any value.
+  SANITIZER="default"
```

Comment:
oof I wonder if this breaks CIFuzz which uses run_fuzzer: https://github.com/google/oss-fuzz/blob/26e8d7c7728096edf55a1fb6d0ecbc4b2dae6afa/infra/cifuzz/fuzz_target.py#L131
infra-tests didn't run (it makes an incorrect assumption that makes it only run tests when there is a code change to the module a test is for).
Even if it runs, I'm not sure it will test this properly since it may use the old base-runner image.

Reply:
Actually, I guess it won't break it since CIFuzz passes the sanitizer in: https://github.com/google/oss-fuzz/blob/26e8d7c7728096edf55a1fb6d0ecbc4b2dae6afa/infra/cifuzz/fuzz_target.py#L126

Reply:
yeah, seems like `reproduce` is the only path to get there without `SANITIZER` and it is so because I'm only enabling this path with this PR